solidity/random-beacon/test/fixtures/index.ts

```diff
@@ -85,8 +87,51 @@ export async function randomBeaconDeployment(): Promise<DeployedContracts> {
   return contracts
 }
 
+export async function randomBeaconDeploymentWithRealSortitionPool(): Promise<DeployedContracts> {
```

Comment:
Done in https://github.com/keep-network/keep-core/pull/2684/commits/9d8bd07f2914532fd922d9d34442bf83ddfa680e.

There is only one `randomBeaconDeployment` function now, but it would be cleaner if we just got rid of `SortitionPoolStub` completely in the future.